localstack/services/kinesis/kinesis_listener.py

```diff
@@ -304,6 +292,29 @@ def decode_content(self, data, describe=False):
 
         return decoded
 
+    def create_event_record(self, data, sequence_number):
+        # Note: avoid adding 'encryptionType':'NONE' in the event_record, as this breaks .NET Lambdas
+        return {
+            "approximateArrivalTimestamp": epoch_timestamp(),
+            "data": data["Data"],
+            "partitionKey": data["PartitionKey"],
+            "sequenceNumber": sequence_number,
+        }
+
+    def check_for_dynamodb_source_arn(self, event_record):
```

Comment:
this function name is also confusing to me because I would expect it to return a boolean. 
consider renaming it to something like `get_dynamodb_event_source_arn` and making it clear in the docstring that if the event_record is not from dynamodb the function returns `None`